### ISSUE TITLE
Exception in thread "AWT-EventQueue-1" java.lang.UnsupportedClassVersionError on Ubuntu 18.04 + OpenJdk-8

### ISSUE THREAD

**Issue by serpro69** [mguessan/davmail#39]
Getting the following error when running DavMail on Ubuntu 18.04 with OpenJdk-8
```
Exception in thread "AWT-EventQueue-1" java.lang.UnsupportedClassVersionError: javafx/embed/swing/JFXPanel has been compiled by a more recent version of the Java Runtime (class file version 54.0), this version of the Java Runtime only recognizes class file versions up to 52.0
        at java.lang.ClassLoader.defineClass1(Native Method)
        at java.lang.ClassLoader.defineClass(ClassLoader.java:763)
        at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
        at java.net.URLClassLoader.defineClass(URLClassLoader.java:468)
        at java.net.URLClassLoader.access$100(URLClassLoader.java:74)
        at java.net.URLClassLoader$1.run(URLClassLoader.java:369)
        at java.net.URLClassLoader$1.run(URLClassLoader.java:363)
        at java.security.AccessController.doPrivileged(Native Method)
        at java.net.URLClassLoader.findClass(URLClassLoader.java:362)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:349)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        at java.lang.Class.forName0(Native Method)
        at java.lang.Class.forName(Class.java:264)
        at davmail.ui.SettingsFrame.isJavaFXAvailable(SettingsFrame.java:267)
        at davmail.ui.SettingsFrame.getSettingsPanel(SettingsFrame.java:164)
        at davmail.ui.SettingsFrame.<init>(SettingsFrame.java:748)
        at davmail.ui.tray.FrameGatewayTray.createAndShowGUI(FrameGatewayTray.java:324)
        at davmail.ui.tray.FrameGatewayTray$8.run(FrameGatewayTray.java:198)
        at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
        at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
        at java.awt.EventQueue.access$500(EventQueue.java:97)
        at java.awt.EventQueue$3.run(EventQueue.java:709)
        at java.awt.EventQueue$3.run(EventQueue.java:703)
        at java.security.AccessController.doPrivileged(Native Method)
        at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
        at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
        at org.GNOME.Accessibility.AtkWrapper$6.dispatchEvent(AtkWrapper.java:715)
        at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:205)
        at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
        at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
        at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
        at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
        at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by mguessan** [comment]
This is an Ubuntu issue, you are not the first one... you probably have an updated OpenJFX 11 package incompatible with Java 8
=> either downgrade OpenJFX or upgrade OpenJDK

**Comment by serpro69** [comment]
Thanks for the input @mguessan 
I've tried downgrading openjfx to version 8 and it didn't help. Then I purged openjfx with `$ sudo apt-get purge openjfx`, which removed all of it's dependencies and davmail as well. After I explicitly installed openjfx version 8 with `$ sudo apt-get install openjfx=8u161-b12-1ubuntu2 libopenjfx-java=8u161-b12-1ubuntu2 libopenjfx-jni=8u161-b12-1ubuntu2` and then installed davmail. Now everything seems to work (at least I can launch it).
Just in case anyone else will face the same issue.
